backend/src/schema/resolvers/registration.js

```diff
@@ -51,11 +54,14 @@ export default {
           emailAddress.relateTo(user, 'belongsTo'),
           emailAddress.update({ verifiedAt: new Date().toISOString() }),
         ])
+        await createOrUpdateLocations(args.id, args.locationName, session)
         return user.toJson()
       } catch (e) {
         if (e.code === 'Neo.ClientError.Schema.ConstraintValidationFailed')
           throw new UserInputError('User with this slug already exists!')
         throw new UserInputError(e.message)
+      } finally {
+        session.close()
```

Comment:
maybe these make sense here, if so, please explain... 
if not, please open an issue to refactor, then add them there where our future selves can find them more easily

Reply:
The signature of `createOrUpdateLocations` has changed. It is now expecting session and not driver. When I updated I got complaints that a session was already closed. So I re-use the same session and close it at the end.